test/orderbookReserve.js

```diff
@@ -3193,6 +3193,62 @@ contract('OrderbookReserve', async (accounts) => {
             assert.equal(orderList.length, 4);
         })
 
+        it("test can't 'revive' canceled (tok to eth) order by updating it", async() => {
+            let tokenWeiDepositAmount = new BigNumber(500 * 10 ** 18);
```

Comment:
JS nit: it's probably better to use `const` rather than `let` if possible to minimize later assignments by mistake.